### PR TITLE
Cleanup prison_1.json

### PR DESCRIPTION
**Pull request by FarkasKBence**
#### Summary
Content "Minor prison mapgen update"

#### Purpose of change

prison_1.json is one of the largest files in `/mapgen`; it's 1636 lines long, and contains 3 locations, of which 2 are unpalettized.

#### Describe the solution

File changes:
* created a mapgen folder for prisons
* moved `prison_1,json` into the new folder
* split the file into 3, 1 for every prison
* created separate palettes for each prison and made a base palette

Mapgen changes:
* regionalized terrains
* added SUS itemgroups, expanded kitchen loot, overhauled kitchens
* added sinks to cells, trash cans to the land prison and Alcatraz
* added flagpoles to Alcatraz, standardized flag spawns
* overhauled land prison workstation with newer furniture
* added solitary cells and guard breakrooms to land prison
* expanded guard breakroom to Alcatraz
* removed most thin Unicode characters from prison_island
* replaced hardwood flooring with concrete
* moved item groups to a new file
* removed `SIDEWALK` tag from land prison

#### Describe alternatives you've considered

None.

#### Testing

Debug-spawned the 4 prisons. Peeked under furniture to check if they're on appropriate terrains.

#### Additional context

The palettes could've been optimized a bit more, but I was too tired to further mess with them.

For a follow-up PR: the watchtowers need to be a bit taller, and the land prison's road connection needs fixing, also the reception area should be expanded upon.

### CODE REVIEW

**Comment by oosyrag** [comment]
Good stuff, I've also been eyeing the prison map. Either change the walls from concrete to fences or add a new low security version. All the prisons #I've seen in my area aren't built like castles, just fence enclosed campuses.

**Review by Maleclypse — APPROVED** [review]
Awesome work.  No strings touched either.

**Comment by Maleclypse** [comment]
Error not related to your changes that I can find.  Test kicked. ```ERROR : src/item.cpp:1501 [ret_val<void> item::put_in(const item &, item_pocket::pocket_type, const bool)] tried to put an item (foodperson_mask) count (1) in a container (backpack) that cannot contain it: pockets unacceptable because item is too long, item is too short, item too big, or not enough space5.570 s: shot_features```

**Comment by LovamKicsiGazsii** [comment]
> Thinking about changing the walls from concrete to fences

I was considering that also. Walls are used here because of the area constraint. We can replace them with chainlink once the outer perimeter is expanded, but that is out of scope for this PR.

**Comment by oosyrag** [comment]
Also prison faction camp

**Comment by Maleclypse** [comment]
> Also prison faction camp

Should be a separate PR.

File: data/json/mapgen/prison/prison_island.json [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+        "`│''  |°°°°°°°°°°°°|''}G....0|^^^^|x,,,,y|,h,h;;h|^^^^||@||||^^^^^^^^^˘˘^^^|....||?|||||'|<¢¢¢|  ' │````````````````````",
+        "`│''  g<°°°nnnn°°°°@''||....0|^^^^|,,,,,,|,,,,;;,|^^^^|%...6|^^^^^^^^^˘˘^^^?....|¢¢¢¢¥¢|'|||?||  ' │````````````````````",
+        "`│''  g<°°°üüÜÜ°°°°|'a||00000|^^^^|DDDWWW|,s,s;;s|^^^^|...h6|^^^^^^^^^˘˘^^^|rrrr|ооĐĐЁ£|''''''f  ' │````````````````````",
+        "`│''  |°°°°°°°°°°°°|||||||||||^^^^|||||||||h,h;;h|^^^^|А1ddd|||||Ĝ||||||@|||||||||||||||Hffffff  ' │````````````````````",
+        "`│''  |eeeeööÖÖóóóó|b|b|b|b|b|^^^^|АAAAooo|||||||||Ĝ||||||gg|bS|,,,|Sb|Y..А|S44|%.dq|/  '        ' │````````````````````",
+        "`│''||||||||||||||||bobobobob|^^^^|,,,,,,,Г|bST|.....|TSb|  |b,Ĝ,,,Ĝ,b|¤.......?..dГg'''''''''’’'' │````````````````````",
+        "`│''|U|U|U|U|З...dd|,,;;,,,,,|^^^^?,,ddd,,,|b,;Ĝ.....Ĝ,,b|  |T,|,,,|,T|¤.......|....|' |w||w||w||||│````````````````````",
+        "`│''|H|H|H|H|1....h|T,;BBBB,,Ĝ^^^^|%,,h,,,Г|||||.....|||||  ||||,,,||||=||=||=||m.1||' |C$|C$|C$|Кй│````````````````````",
+        "`│''g..............|,,;;,,,,,|^^^^||||||||||bST|.....|TSb|  |bS|,,,|Sb|;c|,c|,c|m.З|'' |C.|C.|C¤|ББ│````````````````````",
```
Comment (Night-Pryanik) [inline data/json/mapgen/prison/prison_island.json:74]:
```suggestion
        "`│''g..............|,,;;,,,,,|^^^^||||||||||bST|.....|TSb|  |bS|,,,|Sb|;C|,C|,C|m.З|'' |C.|C.|C¤|ББ│````````````````````",
```
Please change this as `c` contains items from `animalshelter_utility` item group, which is inappropriate for infirmary. Error from my side here.

Comment (Night-Pryanik) [inline data/json/mapgen/prison/prison_island.json:74]:
```suggestion
        "`│''g..............|,,;;,,,,,|^^^^||||||||||bST|.....|TSb|  |bS|,,,|Sb|;C|,C|,C|m.З|'' |C.|C.|C¤|ББ│````````````````````",
```
Please change this as `c` contains items from `animalshelter_utility` item group, which is inappropriate for infirmary. Error from my side here.

Comment (Night-Pryanik) [inline data/json/mapgen/prison/prison_island.json:74]:
```suggestion
        "`│''g..............|,,;;,,,,,|^^^^||||||||||bST|.....|TSb|  |bS|,,,|Sb|;C|,C|,C|m.З|'' |C.|C.|C¤|ББ│````````````````````",
```
Please change this as `c` contains items from `animalshelter_utility` item group, which is inappropriate for infirmary. Error from my side here.

**Pull request merged**